src/parser/priest/shadow/modules/talents/VoidTorrent.tsx

```diff
@@ -73,31 +109,51 @@ class VoidTorrent extends Analyzer {
     this._previousVoidTorrentCast = null;
   }
 
-  onVoidTorrentDamage(event: DamageEvent) {
+  onDamage(event: DamageEvent) {
     this.damage += event.amount || 0;
```

Comment:
`this.damage += event.amount + (event.absorb || 0);`

Reply:
Updated this one with `absorbed` as well like above, if the `absorb` is indeed correct then I can switch it over to that, but based on searching around `absorbed` looks correct.

Reply:
hmmm ... i was looking at some damage events recently and the property was called absorb. ill double check ... maybe something changed

Reply:
Its absorb
![image](https://user-images.githubusercontent.com/1304554/103249993-67e4aa00-4937-11eb-9f3d-979f35d7cae3.png)